src/models/link.es6.js

```diff
@@ -82,23 +82,6 @@ class Link extends Base {
     );
   }
 
-  get thumbnail () {
-    var props = this.props;
-    var thumbnailSrc = '/img/default.gif';
-
-    if (props.thumbnail) {
-      if (props.thumbnail === 'default' || props.thumbnail === 'self' || props.thumbnail === 'nsfw') {
```

Comment:
Thinking it through further, it might be best to have this return `undefined` in case the `props.thumbnail` property isn't a full URL to prevent some of this weirdness sneaking through.